# split up the seconds from the milliseconds since the M4 can't support 64 bit types

uint32 seconds
uint32 millis